From 875c449cea26bbd35b11316fcbd3c53d22c98e9d Mon Sep 17 00:00:00 2001
From: Alexey Brodkin <abrodkin@synopsys.com>
Date: Tue, 15 Jan 2019 16:18:39 +0300
Subject: [PATCH] ARC: [hsdk]: Enable U-Boot support

This is required to get from U-Boot either
customized kernel command line of even entire .dtb

Upstream status: Inappropriate.

Signed-off-by: Alexey Brodkin <abrodkin@synopsys.com>
---
 arch/arc/configs/hsdk_defconfig | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arc/configs/hsdk_defconfig b/arch/arc/configs/hsdk_defconfig
index 2a1d2cbfee1a..ef9b90550b18 100644
--- a/arch/arc/configs/hsdk_defconfig
+++ b/arch/arc/configs/hsdk_defconfig
@@ -77,3 +77,5 @@ CONFIG_DEFAULT_HUNG_TASK_TIMEOUT=10
 # CONFIG_DEBUG_PREEMPT is not set
 # CONFIG_FTRACE is not set
 CONFIG_CRYPTO_ECHAINIV=y
+# CONFIG_ARC_SMP_HALT_ON_RESET is not set
+CONFIG_ARC_UBOOT_SUPPORT=y
-- 
2.16.2

